doc/developers/contributing.rst

```diff
@@ -619,16 +619,30 @@ Finally, follow the formatting rules below to make it consistently good:
         of the mentioned shapes. The default value is
         `np.ones(shape=(n_samples,))`.
 
+    list_param : list of int
+
+    typed_ndarray : ndarray of shape (n_samples,), dtype=np.int32
+
+    sample_weight : array-like of shape (n_samples,), default=None
+
 In general have the following in mind:
 
     1. Use Python basic types. (``bool`` instead of ``boolean``)
-    2. Use parenthesis for defining shapes: ``array-like of shape (n_samples,)``
-       or ``array-like of shape (n_samples, n_features)``
+    2. Use parenthesis for defining shapes: ``array-like, shape=(n_samples,)``
+       or ``array-like, shape=(n_samples, n_features)``
```

Comment:
I think I missed the point where we decided to go back to using this syntax, where did that happen?